### ISSUE TITLE
Add bracket pair colourization

### ISSUE THREAD

**Issue by john-sandall** [CoefficientSystems/coefficient-cookiecutter#33]
Uninstall Bracket Pair Colorizer (1 & 2), VS Code has this functionality built-in, enable via `settings.json`

```
{
    "editor.bracketPairColorization.enabled": true,
    "editor.guides.bracketPairs":"active"
}
```

https://github.com/CoenraadS/Bracket-Pair-Colorizer-2#readme

**Comment by hicksl** [comment]
https://github.com/CoefficientSystems/coefficient-cookiecutter/pull/59

**Comment by john-sandall** [comment]
Closed by #59
